### ISSUE TITLE
uiSchema `accept` is not applied except while using the default theme

### ISSUE THREAD

**Issue by samialitop3** [rjsf-team/react-jsonschema-form#3577]
### Prerequisites

- [X] I have searched the existing issues
- [X] I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [X] I have read the [documentation](https://rjsf-team.github.io/react-jsonschema-form/docs)
- [X] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net), [Codesandbox.io](https://codesandbox.io) or preferably a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### What theme are you using?

material-ui

### Version

5.5.0

### Current Behavior

The `accept` option is not applied to the input using any theme except the default theme. 
Other options are applicable.

### Expected Behavior

When passing the `accept` option it should add the `accept=".fileType"` to the input element.

### Steps To Reproduce

This can be found on the playground link and has proven to be broken except on the default theme.
https://rjsf-team.github.io/react-jsonschema-form/

### Environment

```markdown
- OS: macOS
- Node: v16.19.1
- npm: 8.19.3
```

**Comment by heath-freenome** [comment]
Is this for files?

**Comment by samialitop3** [comment]
Yes, this is for files that are defined in the schema like this
`file_element: {
    type: 'string',
    format: 'data-url',
}`

and in the uiSchema when defining the `ui:options` to be `{ accept: '.pdf` }`}, it works only with the default theme, all other themes don't work.

**Comment by heath-freenome** [comment]
@samialitop3 AH, yeah, this is an issue for many of the themes. I think that the material-ui themes might work given how they are implemented.

**Comment by samialitop3** [comment]
@heath-freenome The material-ui theme is not working as well, you can check it on the playground link

**Comment by a3957273** [comment]
Looks like 'multiple' also doesn't work on MUI v5.  Not sure whether to create a new error or include it in this one.

**Comment by heath-freenome** [comment]
> Looks like 'multiple' also doesn't work on MUI v5. Not sure whether to create a new error or include it in this one.

Keep it in this one

**Comment by samialitop3** [comment]
@heath-freenome Any updates on this?

**Comment by heath-freenome** [comment]
@samialitop3 Given we have a few, very busy volunteers maintaining the project, we rely largely on contributions from people who are willing to provide fixes/features. Are you able to help on this?

**Comment by heath-freenome** [comment]
@samialitop3 It looks like this has been fixed in v6. If you use this [playground](https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6e30sInNjaGVtYSI6eyJ0aXRsZSI6IkZpbGVzIiwidHlwZSI6Im9iamVjdCIsInByb3BlcnRpZXMiOnsiZmlsZSI6eyJ0eXBlIjoic3RyaW5nIiwiZm9ybWF0IjoiZGF0YS11cmwiLCJ0aXRsZSI6IlNpbmdsZSBmaWxlIn0sImZpbGVzIjp7InR5cGUiOiJhcnJheSIsInRpdGxlIjoiTXVsdGlwbGUgZmlsZXMiLCJpdGVtcyI6eyJ0eXBlIjoic3RyaW5nIiwiZm9ybWF0IjoiZGF0YS11cmwifX0sImZpbGVzQWNjZXB0Ijp7InR5cGUiOiJzdHJpbmciLCJmb3JtYXQiOiJkYXRhLXVybCIsInRpdGxlIjoiU2luZ2xlIEZpbGUgd2l0aCBBY2NlcHQgYXR0cmlidXRlIn19fSwidWlTY2hlbWEiOnsiZmlsZXNBY2NlcHQiOnsidWk6b3B0aW9ucyI6eyJhY2NlcHQiOiIucGRmIn19fSwidGhlbWUiOiJkYWlzeS11aSIsImxpdmVTZXR0aW5ncyI6eyJzaG93RXJyb3JMaXN0IjoidG9wIiwidmFsaWRhdGUiOmZhbHNlLCJkaXNhYmxlZCI6ZmFsc2UsIm5vSHRtbDVWYWxpZGF0ZSI6ZmFsc2UsInJlYWRvbmx5IjpmYWxzZSwib21pdEV4dHJhRGF0YSI6ZmFsc2UsImxpdmVPbWl0IjpmYWxzZSwiZXhwZXJpbWVudGFsX2NvbXBvbmVudFVwZGF0ZVN0cmF0ZWd5IjoiY3VzdG9tRGVlcCIsImV4cGVyaW1lbnRhbF9kZWZhdWx0Rm9ybVN0YXRlQmVoYXZpb3IiOnsiYXJyYXlNaW5JdGVtcyI6InBvcHVsYXRlIiwiYWxsT2YiOiJza2lwRGVmYXVsdHMiLCJjb25zdEFzRGVmYXVsdHMiOiJhbHdheXMiLCJlbXB0eU9iamVjdEZpZWxkcyI6InBvcHVsYXRlQWxsRGVmYXVsdHMiLCJtZXJnZURlZmF1bHRzSW50b0Zvcm1EYXRhIjoidXNlRm9ybURhdGFJZlByZXNlbnQifSwidXNlRmFsbGJhY2tGaWVsZCI6ZmFsc2UsInVzZUZhbGxiYWNrVWlGb3JVbnN1cHBvcnRlZFR5cGUiOmZhbHNlLCJsaXZlVmFsaWRhdGUiOmZhbHNlfSwidmFsaWRhdG9yIjoiQUpWOCIsInNhbXBsZU5hbWUiOiJGaWxlcyJ9) you can see that.

**Comment by samialitop3** [comment]
Oh, finally a response after 2 years!

**Comment by heath-freenome** [comment]
@samialitop3 Yeah there are only 2 people maintaining RJSF on a regular basis. With the recent 6.x release, we are revisiting old issues to see if they are still valid. Sorry for the slowness
